### ISSUE TITLE
exporting gzipped table prepends ungzipped header

### ISSUE THREAD

**Issue by pocin** [keboola/sapi-python-client#39]
`client.tables.export_to_file('in.c-test.sample_table', 'out/tables', is_gzip=True)`

results in something like this:

```
"new_opt_account_vwocategorypicklist","ema_IsfromSuite"
?ٿ&��$������K���uR�IQ�?�dB�e4>x�Ң��x�|>���2YΆTN|�<'@��9����Y����^X�� jv
```

ungzipping the output table without header works fine. What are the options here?

**Comment by odinuv** [comment]
altenatives are - support compression https://github.com/keboola/sapi-python-client/issues/26 or is_gzip false

**Comment by pocin** [comment]
So the api never returns the table contents with a header?

If not, we probably have to ungzip the table on the fly and gzip back again https://docs.python.org/3/library/gzip.html ?

**Comment by odinuv** [comment]
yes, the exported table data (as available in the storage files) never contains the header. So the file has to be unzipped first and then the headers must be added. And yes - it's two things - whether the file should be transferred gziped (should be on by default, but is off, as python clien does not support it yet) and whether the result should be gzipped (would need recompressing as you described). The critical thing here is that the gzip implmenetation must not load the file in the memory, which needs some testing.

**Comment by odinuv** [comment]
fixed in 8b41ada8023001abb5f3e53bffd3ce8bc7189378
